src/js/index.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+// test
+
+const $total = document.querySelector("#total");
+const $digits = document.querySelector(".digits");
+const $operations = document.querySelector(".operations");
+const $modifier = document.querySelector(".modifier");
+
+let isOperationClicked = false;
+let isOperator2In = false; // 두번째 인자가 들어왔는지 체크
+let operator1 = 0;
+let operator2 = 0;
+let operation = "";
+let result = 0;
+
+const initCalculator = () => {
+  isOperationClicked = false;
+  isOperator2In = false;
+  operator1 = 0;
+  operator2 = 0;
+  operation = "";
+  result = 0;
+  $total.textContent = "0";
+};
+
+// 결과창에 보여줄 숫자 제어
+const showTotalNumber = (currentTotal, nextNumber) => {
+  let newTotal = "";
+
+  if (isOperationClicked && !isOperator2In) {
+    newTotal = nextNumber;
+    isOperator2In = true;
+  } else if (isOperationClicked && isOperator2In) {
+    newTotal = currentTotal + nextNumber;
+  } else {
+    if (currentTotal === "0") {
+      newTotal = nextNumber;
+    } else {
+      newTotal = currentTotal + nextNumber;
+    }
+  }
+
+  return newTotal;
+};
```

Comment:
```suggestion
const showTotalNumber = (currentTotal, nextNumber) => {
  if (isOperationClicked && !isOperator2In) {
    isOperator2In = true
    return nextNumber
  }
  if (isOperationClicked && isOperator2In) {
    return currentTotal + nextNumber
  }
  if (currentTotal === '0') {
    return nextNumber
  }
  return currentTotal + nextNumber
}
```

newTotal이라는 지역변수를 활용하는것도 좋지만 아무래도 코드의 흐름을 읽어야 하는 불편함이 있는것 같아요. 
early return 활용하여 계산이 끝나면 바로 반환하도록 하였고 else if와 else의 사용을 줄여보았습니다.

Reply:
이 부분은 생각지도 못한 부분입니다! 🙏 새로 배워가요 ㅎㅎ